package.json

```diff
@@ -3,15 +3,16 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
+    "prebuild": "rm -rf node_modules/.cache",
     "check:node": "check-node-version --node \">=8.1 <9\"",
     "test": "npm run lint",
-    "dev": "bin/buildStatic.sh && next dev",
+    "dev": "npm run prebuild && bin/buildStatic.sh && next dev",
     "build": "bin/buildStatic.sh && next build",
```

Comment:
prebuild should happen on "build" script. Why not add the `bin/buildStatic.sh` to prebuild and then just call prebuild in both places.